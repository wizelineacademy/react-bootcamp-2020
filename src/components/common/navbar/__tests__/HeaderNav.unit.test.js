import React from 'react';
import {shallow} from 'enzyme';
import HeaderNav from '../Navbar';

test('renders HeaderNav', () => {
  const wrapper = shallow(
    <HeaderNav />
  );
  expect(wrapper).toMatchSnapshot();
});