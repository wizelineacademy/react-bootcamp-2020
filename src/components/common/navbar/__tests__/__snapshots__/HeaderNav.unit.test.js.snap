// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders HeaderNav 1`] = `
<Menu
  borderless={true}
  className="top-menu"
  fixed="top"
>
  <MenuItem
    className="logo"
    header={true}
  >
    <i
      className="fab fa-react"
    />
    <p
      className="name-header"
    >
      React bootcamp 2020
    </p>
  </MenuItem>
  <MenuMenu
    className="nav-container"
  >
    <SearchBar />
  </MenuMenu>
  <MenuMenu
    position="right"
  >
    <MenuItem
      onClick={[Function]}
    >
      <Icon
        as="i"
        className="header-icon"
        name="sun"
        size="large"
      />
    </MenuItem>
    <MenuItem>
      <Icon
        as="i"
        className="header-icon"
        name="home icon"
        size="large"
      />
    </MenuItem>
    <MenuItem>
      <Icon
        as="i"
        className="header-icon"
        name="star"
        size="large"
      />
    </MenuItem>
    <MenuItem
      name="avatar"
    >
      <Image
        as="img"
        avatar={true}
        size="tiny"
        src="https://sites.google.com/site/heroespoderosos/_/rsrc/1397058950773/home/Estatua_de_cera_de_Luffy.png?height=330&width=400"
        ui={true}
      />
    </MenuItem>
  </MenuMenu>
</Menu>
`;
